resources/overrides/library/sap.ui.layout.d.ts

```diff
@@ -125,3 +125,167 @@ declare module "sap/ui/layout/SideContentVisibility" {
+}
+
+declare module "sap/ui/layout/form/ColumnCells" {
+	import {form} from "sap/ui/layout/library";
+
+	/**
+	 * An <code>int</code> type that defines how many cells a control inside of a column of a <code>Form</code> control using the <code>ColumnLayout</code> control as layout can use.
+	 * 
+	 * Allowed values are numbers from 1 to 12 and -1. -1 means the value is calculated.
+	 *
+	 * @public
+	 * @since 1.56.0
+	*/
+	export default form.ColumnCells;
+}
+
+declare module "sap/ui/layout/form/ColumnsL" {
+	import {form} from "sap/ui/layout/library";
+
+	/**
+	 * An <code>int</code> type that defines how many columns a <code>Form</code> control using the <code>ColumnLayout</code> as layout can have if it has large size
+	 * 
+	 * Allowed values are numbers from 1 to 3.
+	 *
+	 * @public
+	 * @since 1.56.0
+	*/
+	export default form.ColumnsL;
+}
+
+declare module "sap/ui/layout/form/ColumnsM" {
+	import {form} from "sap/ui/layout/library";
+
+	/**
+	 * An <code>int</code> type that defines how many columns a <code>Form</code> control using the <code>ColumnLayout</code> as layout can have if it has medium size
+	 * 
+	 * Allowed values are numbers from 1 to 2.
+	 *
+	 * @public
+	 * @since 1.56.0
+	*/
+	export default form.ColumnsM;
+}
+
+declare module "sap/ui/layout/form/ColumnsXL" {
+	import {form} from "sap/ui/layout/library";
+
+	/**
+	 * An <code>int</code> type that defines how many columns a <code>Form</code> control using the <code>ColumnLayout</code> as layout can have if it has extra-large size
+	 * 
+	 * Allowed values are numbers from 1 to 6. <b>Note:</b> In versions lower than 1.89 only 4 columns are allowed.
+	 *
+	 * @public
+	 * @since 1.56.0
+	*/
+	export default form.ColumnsXL;
+}
+
+declare module "sap/ui/layout/form/EmptyCells" {
+	import {form} from "sap/ui/layout/library";
+
+	/**
+	 * An <code>int</code> type that defines how many cells beside the controls inside of a column of a <code>Form</code> control using the <code>ColumnLayout</code> control as layout are empty.
+	 * 
+	 * Allowed values are numbers from 0 to 11.
+	 *
+	 * @public
+	 * @since 1.56.0
+	*/
+	export default form.EmptyCells;
+}
+
+declare module "sap/ui/layout/form/GridElementCells" {
+	import {form} from "sap/ui/layout/library";
+
+	/**
+	 * A string that defines the number of used cells in a <code>GridLayout</code>. This can be a number from 1 to 16, "auto" or "full". If set to "auto" the size is determined by the number of fields and the available cells. For labels the auto size is 3 cells. If set to "full" only one field is allowed within the <code>FormElement</code>. It gets the full width of the row and the label is displayed above. <b>Note:</b> For labels full size has no effect.
+	 *
+	 * @deprecated (since 1.67.0) - as <code>sap.ui.commons</code> library is deprecated and the <code>GridLayout</code> must not be used in responsive applications. Please use <code>ResponsiveGridLayout</code> or <code>ColumnLayout</code> instead.
```

Comment:
```suggestion
	 * @deprecated (since 1.67.0) - as the <code>sap.ui.commons</code> library is deprecated and the <code>GridLayout</code> must not be used in responsive applications. Please use <code>ResponsiveGridLayout</code> or <code>ColumnLayout</code> instead.
```